tests/scripts/configure_windows_instance.bat

```diff
@@ -0,0 +1,10 @@
+call .\tests\scripts\install_chocolatey.bat
```

Comment:
Will it be possible to add some docs/headers about how this particular script works, what arguments are required and what requirements are needed?